components/brave_rewards/resources/extension/brave_rewards/_locales/en_US/messages.json

```diff
@@ -214,7 +214,7 @@
     "description": "We show this string in the notification when you don't have enough funds for contribution"
   },
   "contributeNotificationTipError": {
-    "message": "There was a problem processing your tip, please try again.",
+    "message": "Unable to send your tip. Please try again later.",
```

Comment:
We can skip this change in 0.60, so that we don't trigger new translations

Reply:
yup, that sounds good! @NejcZdovc mind removing the strings moving that change into `0.61.x` instead? This way we can get this uplifted into `0.60.x` and the string changes can go into `0.61.x`.